src/ActionModal/components/ActionModal.vue

```diff
@@ -502,9 +502,16 @@ export default {
       return this.featureFlag === "undelegate" ? 0 : this.amount
     },
     invoiceTotal() {
+      // emoney-mainnet and kava-mainnet don't allow discounts on fees
       if (
-        Number(this.subTotal) + this.estimatedFee >
-        this.selectedBalance.amount
+        this.networkId === "emoney-mainnet" ||
+        this.networkId === "kava-mainnet"
+      ) {
+        return Number(this.subTotal) + this.estimatedFee
+      }
+      if (
+        this.gasEstimate &&
+        Number(this.subTotal) + this.estimatedFee > this.selectedBalance.amount
```

Comment:
```suggestion
        Number(this.subTotal) + this.estimatedFee > this.selectedBalance.amount
        // emoney-mainnet and kava-mainnet don't allow discounts on fees
        && this.networkId !== "emoney-mainnet"
        && this.networkId !== "kava-mainnet"
```

that avoids the double code in the beginning of the function

Reply:
yeah, I thought of doing this too, but I thought it would be a too long of an if statement. 

But I can change it

Reply:
I don't mind. pick you.